bugreport_23012023.md

```diff
@@ -0,0 +1,6 @@
+1. After creating the env, notebook can't be moved to another folder
+2. When you create the env in the source folder, you have to change the working directory before you run the jupyter lab
+3. Decimal comma is not recognized
+4. LGC proficiency testing reports require alternative number of decimal points. One have to look for those in .json file
+5. Not sure if Alzymologist Oy Lab is a legal name.
```

Comment:
`Alzymologist Oy` is legal company name. I suppose appending the word "laboratory" in some form is legit language thing?

Reply:
Ok, I guess so.